statsmodels/tools/tests/test_web.py

```diff
@@ -7,23 +7,23 @@
 class TestWeb(TestCase):
     def test_string(self):
         url = _generate_url('arch',True)
-        assert_equal(url, 'http://statsmodels.sourceforge.net/stable/search.html?q=arch&check_keywords=yes&area=default')
+        assert_equal(url, 'http://www.statsmodels.org/stable/search.html?q=arch&check_keywords=yes&area=default')
         url = _generate_url('arch',False)
-        assert_equal(url, 'http://statsmodels.sourceforge.net/devel/search.html?q=arch&check_keywords=yes&area=default')
+        assert_equal(url, 'http://www.statsmodels.org/devel/search.html?q=arch&check_keywords=yes&area=default')
         url = _generate_url('dickey fuller',False)
-        assert_equal(url, 'http://statsmodels.sourceforge.net/devel/search.html?q=dickey+fuller&check_keywords=yes&area=default')
+        assert_equal(url, 'http://www.statsmodels.org/devel/search.html?q=dickey+fuller&check_keywords=yes&area=default')
 
     def test_function(self):
         url = _generate_url(OLS, True)
-        assert_equal(url, 'http://statsmodels.sourceforge.net/stable/generated/statsmodels.regression.linear_model.OLS.html')
+        assert_equal(url, 'http://www.statsmodels.org/stable/generated/statsmodels.regression.linear_model.OLS.html')
         url = _generate_url(OLS, False)
-        assert_equal(url, 'http://statsmodels.sourceforge.net/devel/generated/statsmodels.regression.linear_model.OLS.html')
+        assert_equal(url, 'http://www.statsmodels.org/devel/generated/statsmodels.regression.linear_model.OLS.html')
 
     def test_nothing(self):
         url = _generate_url(None, True)
-        assert_equal(url, 'http://statsmodels.sourceforge.net/stable/')
+        assert_equal(url, 'http://www.statsmodels.org/stable/')
         url = _generate_url(None, False)
-        assert_equal(url, 'http://statsmodels.sourceforge.net/devel/')
+        assert_equal(url, 'http://www.statsmodels.org/devel/')
```

Comment:
Whoops, didn't notice that. I'll submit a PR with a symlink for `dev` -> `devel`, so that the old links are still the correct way.